.github/workflows/performance.yml

```
@@ -0,0 +1,211 @@
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: comment
+            });
+          }
+          
+    - name: Store baseline on main
+      if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+      run: |
+        cp test_results/performance/*.json performance_baseline.json || true
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)